packages/expo-image/expo-image.podspec

```diff
@@ -29,5 +29,6 @@ Pod::Spec.new do |s|
   s.dependency 'React'
 
   s.dependency 'SDWebImage', sd_web_image_version
+  s.dependency 'SDWebImageSVGKitPlugin', '~> 1.2'
```

Comment:
Shall we make this version also configurable using `sd_web_image_svg_version` ?

Reply:
Also, do you think it would be possible to make this an optional dependency? E.g. only import it when the OS target is iOS <= 12 (or when not specified).

Reply:
1. I think it may be difficult to add this dependency conditionally, from my short research I think Cocoapods don't support that. One of the simplest reasons is that podspecs are serializable to a JSON format.
2. What we could do is to add both dependencies and then register them depending on the OS version.
3. What I was afraid of, were bugs like "this image is rendered <this way> on iOS 13 and <other way> on iOS 12", which are difficult to debug, explain and guard against by developers.
4. On the other hand, the official `SDWebImageSVGCoder` plugin README states that it should be supported — [Backward Deployment](https://github.com/SDWebImage/SDWebImageSVGCoder#backward-deployment) — if so, maybe that should be the way to go?

I will try to add it the way described in the coder's readme and test the rendering differences for a couple of SVGs — if there will be no differences I think we can go with it! 👍 Thanks for pointing that out.

Reply:
Okay gotcha, lets include it 👍

Reply:
Ok, so:
- I tried adding `SDWebImageSVGCoder` as a regular dependency — error, this pod requires a higher deployment target
- then I tried adding it inside a subspec — same error
- turns out `SDWebImageSVGCoder` requires a minimal deployment target of iOS 13.0, i.e. installing `Podfile` has to have `platform :ios, '13.0'` defined (which is not the case by default in Expo)
- the integration we can use is implemented in c92f445d99b9027a95beaa728b1b421bc3a47320, so:
  - do not depend explicitly anywhere on the conflicting pod
  - if it is included, use it when applicable
  - let the users optionally include the `SVGCoder` themselves (by adjusting the `Podfile`'s `platform :ios` and adding the dependency

Reply:
Hmm, that is unfortunate indeed. This makes the Pod a lot less usable I guess as most people will still be targeting lower iOS versions.

I like how the `.m` import code looks. Perhaps not usable in Expo for now, but people in Bare would be able to use it. I guess we'll need to add some documentation describing this later on.